docs/pt/docs/tutorial/cookie-params.md

```diff
@@ -0,0 +1,33 @@
+# Parâmetros de Cookie
+
+Você pode definir parâmetros de Cookie da mesma maneira que define paramêtros com `Query` e `Path`.
```

Comment:
```suggestion
Você pode definir parâmetros de Cookie da mesma maneira que define paramêtros de `consulta` e `rota`.
```

I believe it's important to keep the same terms as the rest of the docs

Reply:
I have followed this guideline to translate (or not) Path and Query:
 - When the original mentions "query parameters", like in the info box of this page, I translated to the portuguese "parâmetros de consulta".
 - When `Query` or `Path` is mentioned in the original with markdown backticks I maintained as is because in markdown a term wrapped in backticks is supposed to be inline code (in these cases refering to the name of the classes Path or Query).

May be an option is to mention in both ways, but it could be a little bit verbose:

"Você pode definir parâmetros de Cookie da mesma maneira que define parâmetros de consulta (com `Query`) e de rota (com `Path`)"